package.json

```diff
@@ -13,16 +13,39 @@
     "yargs": "^15.0.2"
   },
   "devDependencies": {
+    "@defi-wonderland/smock": "^2.0.3",
+    "@nomiclabs/ethereumjs-vm": "^4.2.2",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-etherscan": "^2.1.6",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@typechain/ethers-v5": "^7.0.1",
+    "@typechain/hardhat": "^2.3.0",
+    "@types/chai": "^4.2.21",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.7.10",
+    "@uniswap/v3-core": "^1.0.0",
+    "@uniswap/v3-sdk": "^3.3.2",
+    "chai": "^4.3.4",
+    "dotenv": "^10.0.0",
+    "ethereum-waffle": "^3.4.0",
+    "ethers": "^5.4.6",
+    "hardhat": "^2.6.2",
     "istanbul": "^0.4.5",
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
-    "jest-junit": "^10.0.0"
+    "jest-junit": "^10.0.0",
+    "jsbi": "^3.2.1",
+    "prettier": "^2.3.2",
+    "ts-node": "^10.2.1",
+    "typechain": "^5.1.2",
+    "typechain-common-abi": "^0.1.2",
+    "typescript": "^4.4.2"
   },
   "scripts": {
```

Comment:
There are a few issues we have with the deployer where we've added scripts to help the deployment process.

For example, the Etherscan API limits the amount of parameter data that can be sent to it to verify a contract. To get around this, we created a helper script that outputs flattened contract code compatible with Etherscan UI for verification. We should definitely add that, and other helper scripts here.

Reply:
The deployment config in `configuration/parameters.js` has been updated with the new V3 pool addresses (`uniswapMarket`). I used the pool with the highest TVL in the case where there's more than one pool (different fees). The REPv2 (Augur) pool seems to have very low liquidity and I've marked it with a comment.